src/clj/puppetlabs/trapperkeeper/rpc/core.clj

```diff
@@ -68,34 +70,62 @@
                            (:client-key ssl-settings)
                            (:client-ca ssl-settings))))
 
+(def https? (comp some? (partial re-find #"^https")))
```

Comment:
This might be a good time to use some interop and just call `.startsWith` on the string.